examples/dynamoathenamigration/migration.go

```diff
@@ -723,7 +723,7 @@ func mergeFiles(files []*os.File, outputFile *os.File) error {
 		}
 
 		nextLine, err := readLine(min.Dec)
-		if trace.Unwrap(err) == io.EOF {
+		if errors.Is(trace.Unwrap(err), io.EOF) {
```

Comment:
I think we don't need to trace.Unwrap before errors.Is or errors.As, they should do the unwrapping by themselves. That said I think it's fine to keep the lines close to the original too.